tests/integration/golang/helpers/s3.go

```diff
@@ -2,13 +2,17 @@ package helpers
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	awsConfig "github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
 	"github.com/rotisserie/eris"
 )
 
+var testBuckets = []string{"bucket1", "bucket2", "bucket3"}
```

Comment:
I know this was copied from the docker compose file, but bucket3 is not used anywhere, so let's retire it

Reply:
10-4

Reply:
done